src/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const RPC_AMOY_DEFAULT = process.env.RPC_AMOY || "https://polygon-amoy.drpc.org"
+export const CONTRACT_REGISTRY_AMOY_DEFAULT = process.env.CONTRACT_REGISTRY_AMOY || "0xBC56d0883ef228b2B16420E9002Ece0A46c893F8"
```

Comment:
I think we should not have default addresses or urls, but rather show an error message in case it is not configured.

Reply:
They are only for unit tests, that default URL is used for a test to create a did and use that network to communicate with the contract. The same with the address registry that I will use to do the tests.

Reply:
Okay